extensions/graphql/src/__tests__/fixtures/graphql-test/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "graphql-test",
```

Comment:
I find it weird to have a standalone package (`graphql-test`) inside `src` folder. Have you considered changing this package into a proper first-class package in our monorepo and moving the files e.g. to `/fixtures`? Considering that this package is also contributing tests you want to run as part of graphql test suite, perhaps the package should live in `/acceptance` or even `/examples` folder?

Reply:
It's a chicken-egg issue here. On one hand, such application serves as both test fixture and example. Moving it alone to a separate package will create circular dependency (the app pkg has dep on `@loopback/graphql`, which has dev dep on the app).

Let me move it to `src/__examples__` for now.

Reply:
I see your point. Here are my concerns:

- If the example includes `package.json`, then we need an automated test verify that the package metadata is valid, this test must be a part of our CI builds. 
- You are introducing a new way for working with test fixtures. This makes maintenance more difficult - maintainers working in the existing code have to learn one more thing, maintainers working on a new feature must decide which fixture style to use.

In general, I think it's a bad idea to use the same code as an example app and for acceptance-level testing. Example apps and test fixtures are serving different kinds of users, they have different requirements. In example apps, the focus is on illustrating the feature and making the design & implementation easy to understand, usually on a very small subset of possible use cases. In test fixtures, we usually want to cover many use cases in a flexible way, often trading ease of learning for better maintainability via code reuse, etc.

Take a look at how we are testing Repository API against connectors: the acceptance tests are not part of `@loopback/repository`, we have test-only projects in `acceptance/repository-*` that are executed as part of our CI build. Similarly we don't have any truly end-to-end test for REST APIs in `@loopback/rest`, we kind of rely on `@examples/todo` and `@examples/todo-list` for acceptance-level tests executing all layers.

I am proposing to rework your example into a proper example project in `examples/*` and break the circular dependency by NOT including example endpoints in GraphQL extension tests. Instead, the example project should come with its own test suite that will be executed as part of `npm test`. If you feel your current extension does not have sufficient test coverage without that example project, then let's find a way how to write unit or integration level tests to change that.